packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -60,7 +60,7 @@ export class BlockList extends Component {
 			renderFooterAppender: this.props.renderFooterAppender,
 			renderAppender: this.props.renderAppender,
 			onDeleteBlock: this.props.onDeleteBlock,
-			contentStyle: this.props.contentstyle,
+			contentStyle: this.props.contentStyle,
```

Comment:
Note: this is an unrelated (bug?) I found that doesn't seem to change anything when changed. However, `contentstyle` (original with no caps) ONLY appears in this one case, while `contentStyle` appears everywhere else (and is the actual prop for the RN `View`).